packages/react/src/jsx/ReactJSXElement.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Would be nice to have `@flow` on this. Not blocking though because we didn't have it before either.

Reply:
I can do this in a separate PR after landing the Babel transform!